static/js/tools/scatter/place_and_type_options.tsx

```diff
@@ -150,14 +172,20 @@ function PlaceAndTypeOptions(props: PlaceAndTypeOptionsProps): JSX.Element {
 
 function loadEnclosedPlaces(
   place: PlaceInfoWrapper,
-  isLoading: IsLoadingWrapper
+  isLoading: IsLoadingWrapper,
+  setEnclosedPlacesError: (failedSettings: {
```

Comment:
please add a comment here to describe the state this is tracking (will be helpful for the future versions of us looking at the code).